ethexe/contracts/src/IRouter.sol

```diff
@@ -56,8 +56,8 @@ interface IRouter {
 
     /// @notice Emitted when validators for the next era has been set.
     /// @dev This is an *informational* and *request* event, signaling that validators has been set for the next era.
-    /// @param startTimestamp timestamp when the new era starts.
-    event NextEraValidatorsCommitted(uint256 startTimestamp);
+    /// @param eraIndex The index of the era for which the validators have been committed.
+    event NextEraValidatorsCommitted(uint256 eraIndex);
```

Comment:
It think naming NextEraValidatorsCommitted with eraIndex is not clear enough
better maybe like this then
```
event ValidatorsCommittedForEra(uint256 eraIndex);
```